lib/pharos/version_command.rb

```diff
@@ -21,15 +21,15 @@ def phases
     end
 
     def addons
-      Pharos::Addon.descendants.sort_by(&:name)
+      Pharos::AddonManager.addons.sort_by(&:name)
     end
 
     def load_phases
       Pharos::PhaseManager.load_phases(__dir__ + '/phases')
     end
 
     def load_addons
-      Pharos::AddonManager.load_addons([__dir__ + '/addons'])
+      Pharos::AddonManager.load_addons([File.join(__dir__, '..', '..', 'addons'),])
```

Comment:
Still failing on rubocop

```
lib/pharos/version_command.rb:32:81: C: Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array. (https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)
      Pharos::AddonManager.load_addons([File.join(__dir__, '..', '..', 'addons'),])
^
```

How does the `load_addons([...])` even work when it's defined as `def self.load_addons(*paths)`? Shouldn't it be `Pharos::AddonManager.load_addons(File.join(...))`, or does ruby somehow implicitly flatten the array?